### DISCUSSION TITLE
How to get short hostname?

### DISCUSSION THREAD

**Discussion by eggbean** [starship/starship#3208] [Q&A] [answered]
Currently I have this in my starship configuration:

```
[hostname]
ssh_only = false
format =  "@[$hostname](bold green):"
trim_at = ".mydomain.co.uk"
disabled = false
```
This is fine until I am using the same config on one of my client servers, as they have different domains. Isn't there an option for short hostname?

**Comment by matchai** [comment, marked as answer]
To trim any domain, you can try using:
```toml
trim_at = "."
```
